typescript/infra/scripts/deploy.ts

```diff
@@ -69,14 +68,28 @@ async function main() {
     deployer = new HyperlaneIsmFactoryDeployer(multiProvider);
   } else if (module === Modules.CORE) {
     config = envConfig.core;
-    const ismFactory = HyperlaneIsmFactory.fromEnvironment(
-      deployEnvToSdkEnv[environment],
-      multiProvider,
-    );
+    // TODO: clean this up
+    let ismFactory: HyperlaneIsmFactory;
+    if (environment === 'test') {
+      const addresses = readJSON(
+        getModuleDirectory(environment, Modules.ISM_FACTORY, context),
+        'addresses.json',
+      );
```

Comment:
So for local deployments it should get the addresses from a local file instead of the SDK, is that right?